src/veryfi/Client.php

```diff
@@ -550,4 +551,297 @@ public function replace_tags(int $document_id,
         return $this->request('PUT', $endpoint_name, $request_arguments);
     }
 
+    /**
+     * Get all W2 documents.
+     *
+     * @param int|null $page The page number, response is capped to a maximum of 50 results per page.
+     * @return string An array of JSON with all W2 documents.
+     * @throws Exception when API version is not supported for W2 documents.
+     */
```

Comment:
Let's add a link to this particular endpoint in docs here and for all others
https://docs.veryfi.com/api/w2s/get-w-2-s/